src/applications/vaos/utils/appointment.jsx

```diff
@@ -67,9 +67,22 @@ export function titleCase(str) {
 }
 
 export function sentenceCase(str) {
-  return `${str.charAt(0).toUpperCase()}${str
-    .substr(1, str.length - 1)
-    .toLowerCase()}`;
+  return str
+    .split(' ')
+    .map((word, index) => {
+      if (/^[^a-z]*$/.test(word)) {
+        return word;
+      }
+
+      if (index === 0) {
+        return `${word.charAt(0).toUpperCase()}${word
+          .substr(1, word.length - 1)
+          .toLowerCase()}`;
+      }
+
+      return word.toLowerCase();
+    })
+    .join(' ');
```

Comment:
This is a fix I forgot to include in a previous PR, which ignores capitalized words when transforming to sentence case.